app/assets/javascripts/discourse/app/helpers/category-link.js

```diff
@@ -7,6 +7,12 @@ import Site from "discourse/models/site";
 import { escapeExpression } from "discourse/lib/utilities";
 import { htmlSafe } from "@ember/template";
 
+const CategoryLink = EmberObject.extend({});
```

Comment:
It doesn't need to be a class does it? The pattern we normally use here is to create a local instance variable:

`let _extraIconRenderers = []` 

It'll be scoped to this module.

Then you can export a function to add a new one to that local variable.